index.js

```diff
@@ -276,6 +276,143 @@ async function getCircuit(conv) {
   }
 }
 
+
+/**
+ * Set a logged on user's presence
+ */
+app.intent('set.presence', async conv => {
+  const circuit = await getCircuit(conv);
+  if (!circuit) {
+    return;
+  }
+  const { presenceType } = conv.contexts.input['setpresence_data'].parameters;
+  const presence = presenceType.toLowerCase();
```

Comment:
Why are you using toLowerCase? I think not using this then down below you can use Circuit.Enums.PresenceState might be better.

Reply:
ignore this comment then